src/components/CustomPressable/index.js

```diff
@@ -19,7 +18,7 @@ const CustomPressable = (props) => {
       onPress={onPress}
       style={style}
       android_ripple={{
-        color: colors.gray1,
+        color: COLORS.balance_gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for the `android_ripple` property has been updated to use `COLORS.balance_gray` from the new `COLORS` object. Please make sure that `COLORS.balance_gray` is the intended color and it matches the previous `colors.gray1` in terms of appearance.
```diff
- color: colors.gray1,
+ color: COLORS.balance_gray,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->